src/watchmaker/status/providers/aws_status_provider.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""AWS Status Provider."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+
+from watchmaker.status.providers.status_provider import (
```

Comment:
Maybe?

```suggestion
from watchmaker.status.providers import (
```

Reply:
On this one, the class in __init__.py is actually status and status_provider is an abstract class for the providers so I renamed to abstract.py instead